package.json

```diff
@@ -106,6 +106,7 @@
     "watch:packages": "python scripts/watch_packages.py"
   },
   "resolutions": {
+    "@codemirror/lang-python": "^6.1.3",
```

Comment:
We should avoid using `resolutions` as much as possible. In this case, could you upgrade the minimal dependency version in [jupyterlab/packages/codemirror/package.json](https://github.com/jupyterlab/jupyterlab/blob/f699d32d19e1687bf27e4c83318f50062adb0c5a/packages/codemirror/package.json#L51) to `^6.1.3`.

```suggestion
```

> For information, if you want to upgrade a third-party dependency in the lock file without changing any package.json, you should use the yarn command [yarn up](https://yarnpkg.com/cli/up).

Reply:
ok, i'll fix it tonight